tests/timelock-controller.tests.ts

```diff
@@ -0,0 +1,166 @@
+import {bnArg, expect, fromSigner, setupContract} from './helpers'
```

Comment:
Overall for tests. Please reformat code with prettier. It will remove all trailing comas and ; in the end, and will format indents